src/cdk/testing/testbed/unit-test-element.ts

```diff
@@ -77,9 +78,23 @@ export class UnitTestElement implements TestElement {
     // supported by mouse events and could lead to unexpected results.
     const clientX = Math.round(left + relativeX);
     const clientY = Math.round(top + relativeY);
+
+    // The latest versions of all browsers we support have the new `PointerEvent` API.
+    // Though since we capture the two most recent versions of these browsers, we also
+    // need to support Safari 12 at time of writing. Safari 12 does not have support for this,
+    // so we need to conditionally create and dispatch these events based on feature detection.
+    const emitPointerEvents = window.PointerEvent !== undefined;
```

Comment:
should this be in cdk/platform? e.g. something like `supportsPointerEvent`

Reply:
Maybe? I could imagine us not doing that since support for pointer events is already quite evolved, and eventually we don't need this anymore. 

If we add it to `cdk/platform` then it would be a public API (unless we decide to prefix it?). I don't feel strongly about it. Let me know what you prefer 😄

Reply:
I'd say probably not- I want to be super conservative with adding new platform apis